client/web/compose/src/components/PageBlocks/RecordConfigurator.vue

```diff
@@ -27,11 +27,86 @@
         style="max-height: 52vh;"
       />
     </b-form-group>
+
+    <div>
+      <b-form-group
+        :label="$t('sanitizers.label')"
```

Comment:
probably not the correct key for the label (guessing its copy pasted)

Reply:
😅 Pilot generated. Will be amended.